README.md

```diff
@@ -184,6 +185,24 @@ Also see our guides for [builds](./docs/builds.md) and [deployments](./docs/depl
 ### How the integration works
 When a workflow (e.g. GitHub Action) or development event (e.g. pull request, commit, branch) runs, the app receives a webhook from GitHub. The app then extract the issue key from the respective branch/commit/PR and send this information to Jira.
 
+## How the backfill works
+The app is designed to backfill historical data into Jira. Once you have installed and configured the app successfully, it will automatically trigger the backfilling process to update Jira with historical information such as pull requests, deployments, branches, builds, and commits.
+
+1. The backfilling process attempts to connect all branches that fulfill at least one of the following criteria:
+    - The branch name contains the issue key.
+    - The title of the latest pull request associated with the branch contains the issue key.
+    - The last commit message of the branch contains the issue key.
```

Comment:
is it just the last commit?  I thought it was like 100....

Reply:
Yeah, I had the same impressions, but that's what the code told us... http://github.com/atlassian/github-for-jira/blob/edcad795ffea87447d305847e460abca3827ee42/src/sync/transforms/branch.ts#L22-L22